src/Stormpath/Cache/MemoryCacheManager.php

```diff
@@ -0,0 +1,10 @@
+<?php namespace Stormpath\Cache;
+
+
+class MemoryCacheManager implements CacheManager {
+
+    public function getCache()
+    {
+        return new MemoryCache();
```

Comment:
What about getting the Cache instance from the cache String key (i.e.: from the region)?